viadot/flows/transform_and_catalog.py

```diff
@@ -0,0 +1,245 @@
+                the `dbt_project.yml` file).
+            dbt_repo_url (str, optional): The URL for cloning the dbt repo with relevant dbt project. Defaults to None.
+            dbt_repo_url_secret (str, optional): Alternatively to above, the secret containing `dbt_repo_url`.
+                Defaults to None.
+            dbt_repo_branch (str, optional): The branch of the dbt repo to use. Defaults to None.
+            token (str, optional): The personal access token. Defaults to None.
+            token_secret (str, optional): The name of the secret storing the token. Defaults to None.
+            local_dbt_repo_path (str, optional): The path where to clone the repo to. Defaults to None.
+            dbt_selects (dict, optional): Valid [dbt node selection](https://docs.getdbt.com/reference/node-selection/syntax)
+                expressions. Valid keys are `run`, `test`, and `source_freshness`. The test select expression is taken
+                from run's, as long as run select is provided. Defaults to None.
+            dbt_target (str): The dbt target to use. If not specified, the default dbt target (as specified in `profiles.yaml`)
+                will be used. Defaults to None.
+            stateful (bool, optional): Whether only the models should be rebuilt only if modified.
+                See [dbt docs](https://docs.getdbt.com/guides/legacy/understanding-state). Defaults to False.
+            metadata_dir_path (Union[str, Path]): The path to the directory containing metadata files.
+                In the case of dbt, it's dbt project's `target` directory, which contains dbt artifacts
+                (`sources.json`, `catalog.json`, `manifest.json`, and `run_results.json`). Defaults to None.
+            luma_endpoint (str, optional): The endpoint of the Luma ingestion API. Defaults to "http://localhost/api/v1/dbt".
+            luma_endpoint_secret (str, optional): The name of the secret storing the luma_endpoint. Defaults to None.
+            vault_name (str, optional): The name of the vault from which to obtain the secrets. Defaults to None.
+
+        Returns:
+            List[str]: Lines from stdout of the `upload_metadata` task as a list.
+
+        Examples:
+            # Build a single model
+            ```python
+            import os
+            from viadot.flows import TransformAndCatalog
+
+            my_dbt_project_path = os.path.expanduser("~/dbt/my_dbt_project")
+
+            flow = TransformAndCatalog(
+                name="Transform and Catalog",
+                dbt_project_path=my_dbt_project_path,
+                dbt_repo_url=my_dbt_repo_url,
+                token=my_token,
+                dbt_selects={"run": "my_model",
+                "source_freshness": "source:schema.table",
+                "test": "my_model"},
+                metadata_dir_path="target",
+                luma_endpoint="https://luma.dyvenia.lan/api/v1/dbt"
```

Comment:
```suggestion
                luma_url="http://localhost"
```
For example, but pls change everywhere